app/controllers/milestones_controller.rb

```diff
@@ -41,9 +41,12 @@ def create
   def update
     respond_to do |format|
       if @milestone.update(milestone_params)
-        Account.where("points >= ?", milestone_params['points']).find_each do |account|
+        User.where("points >= ?", milestone_params['points']).find_each do |account|
```

Comment:
Is there a way to maybe move this query logic into the model rather than the controller? I have done this in the past with the points functionality [here](https://github.com/BennettLai2/WomenInNuclear/blob/main/app/models/event.rb#:~:text=def%20self.,end). This allows us to follow standard procedures for "skinny controller, fat model".